I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 42, in script
    menu:
  File "game/script.rpy", line 43, in <module>
    "Mr Sprinkles Relationship" if mS_Relation == False:
NameError: name 'mS_Relation' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 42, in script
    menu:
  File "D:\renpy-8.0.3-sdk\renpy\ast.py", line 1903, in execute
    choice = renpy.exports.menu(choices, self.set, args, kwargs, item_arguments)
  File "D:\renpy-8.0.3-sdk\renpy\exports.py", line 976, in menu
    condition = renpy.python.py_eval(condition)
  File "D:\renpy-8.0.3-sdk\renpy\python.py", line 1092, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "D:\renpy-8.0.3-sdk\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 43, in <module>
    "Mr Sprinkles Relationship" if mS_Relation == False:
NameError: name 'mS_Relation' is not defined

Windows-10-10.0.22000 AMD64
Ren'Py 8.0.3.22090809
WhoShotMrSprinkles 1.0
Mon Oct 10 19:31:08 2022
